components/Map/zoomTooltip.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '@mui/material/Tooltip';
+
+function ZoomTooltip({ show }) {
+  return (
+    <Tooltip
+      placement="top-end"
+      arrow
+      open={show}
+      title={(
+        <div>
+          <p>
+            <strong>
+              Zoom features are limited while locked into a
+              neighborhood council.
+            </strong>
+            <br />
+            To reset zoom features, please exit by clicking out of the
```

Comment:
We needed to change this to say "please exit by clicking the 'x' on the selected NC" or something like that. More discussion incoming at upcoming general meeting

Reply:
I updated the text to: `To reset zoom features, please exit the boundary selection by clicking the 'X' on the selected Neighborhood Council within the 'Boundaries' filter of the 'Search & Filters' modal.`